### ISSUE TITLE
[Yggtorrent] search is not the same as before

### ISSUE THREAD

**Issue by tchirou** [Jackett/Jackett#11098]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**: Synology docker

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]

**.Net Version**:

**Jackett Version**: 0.17.538

**Last Working Jackett Version**:

**Are you using a proxy or VPN?** no

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]
Hello
Before the url change of yggtorrent, we could manual search any words without minding the order or the partial/complete words
For example, if we queried legacies multi s01e01, we would get "Legacies.​S01E01.​MULTI.​FRENCH.​1080p etc etc". Now it gives zero result. But if we query legacies s01e01 multi, we get "Legacies.​S01E01.​MULTI.​FRENCH.​1080p".
If you could take a look, it would be great !
Thanks in advance

**Comment by ilike2burnthing** [comment]
The site's new search seems to be extremely strict, e.g. `legacies s01e01 multi 1080p hdtv` will return a result, but changing the order or even removing something from the middle like `legacies s01e01 multi hdtv` will return no results.

Putting each keyword in quotations seems to work though, e.g. `"legacies" "s01e01" "multi" "1080p" "hdtv"` and `"legacies" "s01e01" "multi" "hdtv"` both return the same result.

**Comment by tchirou** [comment]
Are you sure ?
Because when I type "legacies" "multi" "s01e01", I don't have any result, but with legacies s01e01 multi, I have one
Thanks a lot

**Comment by ilike2burnthing** [comment]
Sorry, I meant on the site, doing it in Jackett won't work. There will (hopefully) be a fix in the next release that will allow searching for terms normally in any order (you won't need to add quotations).

**Comment by tchirou** [comment]
Great news !
Thanks !

**Comment by tchirou** [comment]
A bit unrelated but as you are already here ;-)
Do you think it will be possible in the future to get a list of results with a checkbox and be able to check the files we want and download them all at once ?

**Comment by ilike2burnthing** [comment]
That would be a quick way to get your IP or account blocked by the site for bot activity (multiple simultaneous downloads, potentially 100 or more if someone ticks them all), so it's not something we'd add.

**Comment by ilike2burnthing** [comment]
[v0.17.546](https://github.com/Jackett/Jackett/releases/tag/v0.17.546)

**Comment by tchirou** [comment]
Thanks a lot, it works
But now, contrary to other indexers like rarbg or sharewood, if we search snowpiercer S01, we won't have all the S01E01, S01E02 and so forth, which is a bummer
Is there any way around that ? 
Many thanks

**Comment by ilike2burnthing** [comment]
This seems to be a restriction of ygg's search (using quotations or not), and I can't find any wildcards which work. If you find a workaround on the site let me know and I'll try to implement in the indexer.

**Comment by WoisWoi** [comment]
Ygg is becoming more and more "unsable"... What are you using as alternative to find good releases in French and Multi (VO + VF) ?

**Comment by tchirou** [comment]
For now, no cloud flare on the new yggtorrent so no need to use flare silver so, so far so good
The alternative is Sharewood

**Comment by WoisWoi** [comment]
> 
> 
> For now, no cloud flare on the new yggtorrent so no need to use flaresolver so, so far so good
> The alternative is Sharewood

Yeah, Flaresolverr is already setup with captcha resolver. But as we all know, the search system of Ygg coupled with Sonarr is a pain in the ass.

**Comment by tchirou** [comment]
I hadn’t had any problem with the former ygg site through jackett and sonarr. 

What you mean about Usenet ? With jackett ? Because sonarr and radarr are compatible with Usenet and there already are Android and iOS apps compatible with Usenet indexers apis.

**Comment by WoisWoi** [comment]
> 
> 
> I hadn’t had any problem with the former ygg site through jackett and sonarr.
> 
> What you mean about Usenet ? With jackett ? Because sonarr and radarr are compatible with Usenet and there already are Android and iOS apps compatible with Usenet indexers apis.

[That's the kind of problem I had](https://www.reddit.com/r/sonarr/comments/l4wll8/sonarr_doesnt_find_all_the_release_of_my_indexers/)
I mean Usenet in general.

**Comment by tchirou** [comment]
Well it is normal
You have French in the sonarr profile, right ?
So sonarr only displays the vostfr and French results, not the multi ones. Multi is considered english as default. In the settings of ygg in jackett, you need to check « Replace MULTI by another language in release name » then below choose « multi.french ». This way sonarr will also displays all the multi results alongside the vostfr and french results.
Problem solved

**Comment by WoisWoi** [comment]
> 
> 
> Well it is normal
> You have French in the sonarr profile, right ?
> So sonarr only displays the vostfr and French results, not the multi ones. Multi is considered english as default. In the settings of ygg in jackett, you need to check « Replace MULTI by another language in release name » then below choose « multi.french ». This way sonarr will also displays all the multi results alongside the vostfr and french results.
> Problem solved

One of my Sonarr setup as example :
![image](https://user-images.githubusercontent.com/75491231/108135725-eb5d8480-70b8-11eb-8670-46942c1ea8a5.png)
![image](https://user-images.githubusercontent.com/75491231/108135784-062ff900-70b9-11eb-8a52-f13b7411ddb6.png)

(I have not setup a "STFR" custom format on this one)

**Comment by tchirou** [comment]
I am not sure I understand what you want to achieve
What kind of files to you want to download ?
Multi and/or vostfr/subfrench/fastsub and/or french ?
Tell me what you want and I will tell you how to get it

**Comment by WoisWoi** [comment]
> 
> 
> I am not sure I understand what you want to achieve
> What kind of files do you want to download ?
> Multi and/or vostfr/subfrench/fastsub and/or french ?
> Tell me what you want and I will tell you how to get it

If possible, i'd like to get MULTI VO (STFR, but I'd like to avoid subtitles inked on the image -> I'm using Bazarr if need be) + VF -> if not possible, then VO (STFR) and in last case the VF

**Comment by ilike2burnthing** [comment]
If you have a Jackett-related issue please open a new issue, otherwise Reddit or another forum may be a better place.

**Comment by tchirou** [comment]
- Set up a language profile with only English and French check, don’t check « upgrade allowed »and save
- set up a release profile with « multi » « vostfr » « subfrench » « fastsub » « french » in the « must contain » box. In the prefered list, give 10 to multi, 9 to vostfr, 8 to subfrench, 7 to fastsub, 6 to French. Write « multi » in the tag box at the end of the page and save
- in the settings of ygg indexer in jackett, check « Replace MULTI by another language in release name » then below choose « multi.french » and save. (It’s optional but nevertheless needed to do if you want only French audio for a specific show, you would only have to use a french as a language profile instead of the English+french one)
- For all the shows you want, use the « english+French » language profile and the « multi » tag in the edit page of the show.

sonarr will download only multi or vostfr or fastsub or subfrench or French files, multi being preferred over the other ones

**Comment by WoisWoi** [comment]
> 
> 
>     * Set up a language profile with only English and French checked, don’t check « upgrade allowed »and save
> 
>     * set up a release profile with « multi » « vostfr » « subfrench » « fastsub » « french » in the « must contain » box. In the prefered list, give 10 to multi, 9 to vostfr, 8 to subfrench, 7 to fastsub, 6 to French. Write « multi » in the tag box at the end of the page and save
> 
>     * in the settings of ygg indexer in jackett, check « Replace MULTI by another language in release name » then below choose « multi.french » and save. (It’s optional but nevertheless needed to do if you want only French audio for a specific show, you would only have to use french as a language profile instead of the English+french one and another release profile with only multi and French, without subfrench and fastsub)
> 
>     * For all the shows you want, use the « english+French » language profile and the « multi » tag in the edit page of the show.
> 
> 
> sonarr will download only multi or vostfr or fastsub or subfrench or French files, multi being preferred over the other ones
> 
> sorry @ilike2burnthing, this was the last one
> @WoisWoi send me a pm if you need

Sorry, but how can I pm on Github :) ? @tchirou

**Comment by tchirou** [comment]
I sent you a pm on Reddit
